src/resqml2_0_1/RockFluidOrganizationInterpretation.cpp

```diff
@@ -0,0 +1,98 @@
+/*-----------------------------------------------------------------------
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"; you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-----------------------------------------------------------------------*/
+#include "resqml2_0_1/RockFluidOrganizationInterpretation.h"
+#include <algorithm>
+
+#include "tools/Misc.h"
+
+using namespace std;
+using namespace epc;
+using namespace RESQML2_0_1_NS;
+using namespace gsoap_resqml2_0_1;
+
+const char* RockFluidOrganizationInterpretation::XML_TAG = "RockFluidOrganizationInterpretation";
+
+RockFluidOrganizationInterpretation::RockFluidOrganizationInterpretation(OrganizationFeature * orgFeat, const std::string & guid, const std::string & title)
```

Comment:
Probably to add a RockFluidUnitInterpretation as another mandatory parameter in this constructor.

Notice that is is to try to go "quickly"!!!!! This is not the long term implementation (several implementations) which should implement the workaround http://docs.energistics.org/#RESQML/RESQML_TOPICS/RESQML-500-106-0-R-sv2010.html (i.e. not really using RockFluidOrganizationInterperation at all but StratigraphicOccurrenceInterpretation instead)